i18n/en.json

```diff
@@ -2992,6 +2992,8 @@
   "create_post.shortcutsNotSupported": "Keyboard shortcuts are not supported on your device.",
   "create_post.write": "Write to {channelDisplayName}",
   "create_team.agreement": "By proceeding to create your account and use {siteName}, you agree to our [Terms of Use]({TermsOfServiceLink}) and [Privacy Policy]({PrivacyPolicyLink}). If you do not agree, you cannot use {siteName}.",
+  "create_team.createTeamRestricted.message": "This is a paid feature, available with a free 30-day trial. Please contact your System Administrator to resolve this.",
+  "create_team.createTeamRestricted.title": "Professional feature",
```

Comment:
Used these title/message to speed up development. @annelieseklein please let us know any updated needed.

Reply:
It seems that this text is shown but doesn't show what feature its hiding. Should we include something in the text that create teams have reached their limit on their current plan or something that is clear what we are referring as a Professional feature. 

@johndavidlugtu any suggestions? (See screenshot in the PR description)